test/functional/apps/discover/group3/_lens_vis.ts

```diff
@@ -655,8 +674,8 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       await discover.waitUntilSearchingHasFinished();
       await testSubjects.missingOrFail('unsavedChangesBadge');
 
-      await discover.chooseLensSuggestion('pie');
-      expect(await getCurrentVisTitle()).to.be('Pie');
+      await discover.chooseLensSuggestion('waffle');
+      expect(await getCurrentVisTitle()).to.be('Waffle');
```

Comment:
After unskipping the test suite and merging main recently, this test started failing. Pie chart suggestion is not available anymore after switching from Bar to Treemap and then to Pie chart as in the test.

cc @elastic/kibana-visualizations